core/src/idx/trees/mtree.rs

```diff
@@ -1406,7 +1407,9 @@ impl Display for MTreeNode {
 		}
 	}
 }
+
 trait NodeVectors: Sized {
+	#[allow(dead_code)]
```

Comment:
I think I am only using it on tests. I will move it to mod test later on, for now that's okay like this